src/module-elasticsuite-core/Search/Request/SortOrder/SortOrderBuilder.php

```diff
@@ -165,7 +165,15 @@ private function addDefaultSortOrders($orders, MappingInterface $mapping)
     private function getSortOrderParams(FieldInterface $field, array $sortOrderParams)
     {
         $sortOrderParams['field']   = $field->getMappingProperty(FieldInterface::ANALYZER_SORTABLE);
-        $sortOrderParams['missing'] = $field->getSortMissing($sortOrderParams['direction']);
+
+        if (empty($sortOrderParams['missing'])
+            || !in_array(
+                    $sortOrderParams['missing'],
+                    [SortOrderInterface::MISSING_FIRST, SortOrderInterface::MISSING_LAST]
+                )
```

Comment:
Can be probably simplified with something along 
```
if (!in_array($sortOrderParams['missing'] ?? false, [SortOrderInterface::MISSING_FIRST, SortOrderInterface::MISSING_LAST]))
```
or 

```
```
if (false === in_array($sortOrderParams['missing'] ?? false, [SortOrderInterface::MISSING_FIRST, SortOrderInterface::MISSING_LAST]))
```
for clarity.
Regards,

Reply:
Thanks. Done.